packages/form-js-editor/assets/form-js-editor-base.css

```diff
@@ -675,6 +677,10 @@
   box-sizing: border-box;
 }
 
+.fjs-properties-container .fjs-properties-panel *:focus-visible {
+  outline: var(--outline-definition);
```

Comment:
In the core styles, we do not show any outline on purpose, but indeed [show a border on focus](https://github.com/bpmn-io/properties-panel/blob/main/src/assets/properties-panel.css#L512). Adding this outline would lead to a too big border.

![image](https://github.com/bpmn-io/form-js/assets/9433996/db3e41ee-8ca2-422a-9ed3-0e34de5ac68a)

We should rather only set the border color here.